### ISSUE TITLE
interferences between ublock Origine and Privacy Badger ?

### ISSUE THREAD

**Issue by walker17x** [gorhill/uBlock#642]
When i use ublocko with privacy badger ( the new privacy plugin from eff ) the web pages loading speed is significantly slowing down.

so if we can not use these two plugins together is it possible to add the privacy badger features in ublocko ?

**Comment by gorhill** [comment]
You are not providing any details, and not providing any hard data (+ methodology) in support of your statement that _"page loading speed is significantly slowing down"_. Also, refer to "Privacy Badger's features" in vague way... what features exactly?

**Comment by harshanvn** [comment]
> add the privacy badger features

I really did not see any benefits of having PB installed on top of uBO (configured in Medium Blocking mode), in my couple of days browsing with it. 

And did not either see any slowdown using it...

**Comment by luxoflux** [comment]
Just for reference, Peter Eckersley who works on Privacy Badger had [this](https://news.ycombinator.com/item?id=10017951) to say about uBlock and PB:

> ublock is great software (it didn't exist when we started work on PB!) and so there are only a few things PB will catch that it doesn't. But there are some. For instance, PB will replace widgets such as Tweet and Facebook like buttons with locally hosted, non-tracking variants. And PB may catch new trackers faster than ublock because it uses algorithmic detection rather than requiring a blocklist.

**Comment by harshanvn** [comment]
> For instance, PB will replace widgets such as Tweet and Facebook like buttons with locally hosted, non-tracking variants.

I think i agree. if there is no filter in this case. it will not be blocked.

> And PB may catch new trackers faster than ublock because it uses algorithmic detection rather than requiring a blocklist.

It depends on how you configure to uBo. In case if its configured > Medium Blocking. I think it will mostly catch the new variants, unless they are locally hosted..

**Comment by gorhill** [comment]
I did [benchmark blocking efficiency of Privacy Badger a long while ago](https://github.com/gorhill/uBlock/wiki/2014-07-22:-%C2%B5Block-and-others:-Blocking-ads,-trackers,-malwares) -- and it was not doing so well back then. I stopped benchmarking PB because I feared I would not be fair to it, as it needs to be "trained" before it blocks something (I did trained it in the benchmark as per PB dev comments somewhere). If I could find a methodology to benchmark properly, I would create new benchmark for blocking efficiency, but I don't have any so far.

Personally, the problem I have as user with PB is the "may catch" (emphasis: _personally, as a user_). uBlock uses well maintained block lists (including HOSTS files), there is no whitelist, and the overall approach in uBlock is that everything out there is a tracker -- a mere IP is what one need for tracking and [**not** connecting to 3rd-party servers](https://github.com/gorhill/uBlock/wiki/Blocking-mode:-medium-mode) by default is a best first-step protection against tracking, no heuristic or filter lists needed.
